include/auxkernels/AuxChemElastic.h

```diff
@@ -29,23 +29,23 @@ class AuxChemElastic : public AuxKernel
 protected:
   virtual Real computeValue();
 
-  virtual Real computeEnergy(Real & conserved, Real & nonconserved, bool matrix);
-  virtual Real computeDifferential(Real & coupled_conserved,Real & nonconserved);
+  virtual Real computeEnergy(const Real & conserved, const Real & nonconserved, bool matrix);
+  virtual Real computeDifferential(const Real & coupled_conserved, const Real & nonconserved);
 
-  virtual Real computeFchem(Real & conserved, Real & nonconserved);
+  virtual Real computeFchem(const Real & conserved, const Real & nonconserved);
   virtual Real computeSelfElasticEnergy(bool matrix);
   virtual Real computeInteractionElasticEnergy(bool matrix);
 
-  virtual Real computeDfchemDcons(Real & coupled_conserved, Real & coupled_nonconserved);
+  virtual Real computeDfchemDcons(const Real & coupled_conserved, const Real & coupled_nonconserved);
   virtual Real computeDselfDcons();
   virtual Real computeDintDcons();
 
-  virtual Real computeDfchemDnoncons(Real & coupled_conserved, Real & coupled_nonconserved);
+  virtual Real computeDfchemDnoncons(const Real & coupled_conserved, const Real & coupled_nonconserved);
```

Comment:
@jwpeterson why did you incorporate this change? Might as well have removed the `&` (or not touched this in this PR)